### ISSUE TITLE
An error occurred while Lint/RequireRelativeSelfPath cop was inspecting ...

### ISSUE THREAD

**Issue by emmahsax** [rubocop/rubocop#10151]
I'm receiving this issue running Rubocop on GitHub Actions:

```
1 error occurred:
An error occurred while Lint/RequireRelativeSelfPath cop was inspecting /home/runner/work/project_name/pproject_name/lib/file_name.rb:10:2.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
https://github.com/rubocop/rubocop/issues

Mention the following information in the issue report:
1.22.0 (using Parser 3.0.2.0, rubocop-ast 1.12.0, running on ruby 3.0.2 x86_64-linux)
```

--------

## Expected behavior

We don't see this error, and Rubocop completes and returns a `0` as the exit code.

## Actual behavior

The error appears, and Rubocop returns an exit code of `1`.

## Steps to reproduce the problem

Probably just need to run Rubocop on GitHub Actions with the versions mentioned in the blurb above.

## RuboCop version

See above.

**Comment by dvandersluis** [comment]
Could you share your code from `/home/runner/work/project_name/pproject_name/lib/file_name.rb` line 10?

**Comment by emmahsax** [comment]
Yes, I think that PR/Issue would resolve this. Do we know when a new tag will be released?

```ruby
# frozen_string_literal: true

require 'yaml'
require 'json'
require 'highline_wrapper'

files = "#{File.expand_path(File.join(File.dirname(File.absolute_path(__FILE__)), 'git_helper'))}/**/*.rb"

Dir[files].each do |file|
  require_relative file
end

module GitHelper; end
```

**Comment by dvandersluis** [comment]
#10143 will fix your issue. Usually new releases come out every 3 weeks or so but we don't have a set schedule. You could always add `# rubocop:disable Lint/RequireRelativeSelfPath` to that line in the meantime.
